libs/sdk-ui-ext/src/internal/interfaces/VisualizationDescriptor.ts

```diff
@@ -60,4 +60,17 @@ export interface IVisualizationDescriptor {
         layoutDescriptor: IFluidLayoutDescriptor,
         settings: ISettings,
     ): IVisualizationSizeInfo;
+
+    /**
+     * Modifies buckets and filters of the insight according to the particular drill down context.
+     *
+     * The exact contract depends on individual {@link IInsight} type, but generally it should replace
+     * the drilled attribute with the Drill Down target target attribute and include the filters from the
+     * drill event into the returned {@link IInsight}.
+     *
+     * @param insight {@link IInsight} to be used for the the drill down application
```

Comment:
This is not going to work, should be `{@link @gooddata/sdk-model#IInsight}` Same with the other occurrence